packages/core/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@prezly/theme-kit-core",
-  "version": "5.3.0",
+  "version": "5.4.1-alpha.0",
```

Comment:
Do I need to revert this before merging or can I merge it with this preview version?